sql/7_0_1-to-7_0_2_upgrade.sql

```diff
@@ -216,3 +216,15 @@ ALTER TABLE `facility` ADD COLUMN `inactive` tinyint(1) NOT NULL DEFAULT '0';
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`) VALUES ('Document_Template_Categories','notification_template','Notification Template',105,0,0,'','','',0,0,1);
```

Comment:
won't the need to be added to database.sql ? (with the 2 current active entries set)

Reply:
Not sure, I didn't touch anything with Facilities?